Packs/FiltersAndTransformers/Scripts/IfElif/README.md

```diff
@@ -0,0 +1,98 @@
+### A transformer for if-elif-else logic.
+
+The **If-Elif** transformer simulates a python *"if elif else"* tree using a JSON provided in the ***conditions*** argument.
+The JSON should be a list of dictionaries where all have the keys "*condition*", which holds a boolean expression, and "*return*", which holds the value to return if "*condition*" is evaluated to be true. To return a default value if all "*condition*"s were false, the last dictionary should have only the key "*default*" holding the valid JSON value. If this is not provided an empty string will be returned as a default.
+In order to prevent injections Context values should be retrieved from the value entered in the ***value (Get)*** of the transformer with the hash-curly brackets `#{...}` syntax. This syntax has the same behavior as the classic XSOAR `${...}` syntax and uses the [Cortex XSOAR Transform Language (DT)](https://xsoar.pan.dev/docs/integrations/dt). To provide the full context to the transformer, use `${.}` as the ***value (Get)*** argument. **Note:** when used as a "*return*" value, this syntax should *not* be surrounded by quotes.  
```

Comment:
```suggestion
In order to prevent injections, context values should be retrieved from the value entered in the *value (Get)* of the transformer with the hash-curly brackets `#{...}` syntax. This syntax has the same behavior as the classic XSOAR `${...}` syntax and uses the [Cortex XSOAR Transform Language (DT)](https://xsoar.pan.dev/docs/integrations/dt). To provide the full context to the transformer, use `${.}` as the *value (Get)* argument. **Note:** when used as a "*return*" value, this syntax should *not* be surrounded by quotes.  
```